src/test/test_dir.c

```diff
@@ -3040,7 +3042,7 @@ vote_tweaks_for_umbw(networkstatus_t *v, int voter, time_t now)
   smartlist_clear(v->supported_methods);
   /* Method 17 is MIN_METHOD_TO_CLIP_UNMEASURED_BW_KB */
   smartlist_split_string(v->supported_methods,
-                         "1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17",
+                         "13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28",
```

Comment:
Should this list also be 25-28?
(See my previous comment)

Reply:
8420898a87f58ddfba28da477465b2f0859870b1 will fix this too.